apps/fluent-tester/src/FluentTester/TestComponents/Common/styles.ts

```diff
@@ -45,6 +45,7 @@ export const commonTestStyles = StyleSheet.create({
     fontSize: 12,
     width: 150,
     marginBottom: 8,
+    paddingVertical: 0,
```

Comment:
is 0 not the default value?

Reply:
No. TextInput has some default padding on Android.

As you can see here the placeholder text appears cut.
<img width="293" alt="Screenshot 2021-06-03 at 9 45 58 PM" src="https://user-images.githubusercontent.com/23552252/120677874-2b544600-c4b5-11eb-8455-bda766f649df.png">